### PR TITLE
Implement Bottom Tab Navigation In Mobile App

### PR DESCRIPTION
**Pull request by rushiii3**
**TabBar preview in android in dark and light mode.**

 

**TabBar preview in ios in dark and light mode.**

 

Fixes [#42](https://github.com/SB2318/UltimateHealth/issues/42)

### CODE REVIEW

File: frontend/src/navigations/TabBar.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            </Pressable>
+          </View>
+        );
+      })}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  mainContainer: {
```
Comment (SB2318) [inline frontend/src/navigations/TabBar.tsx:88]: Reduce Bottom to 0 or 6-7 range.

Set the margin horizontal in a way that the screen becomes within the bottom tab.

Comment (rushiii3) [reply frontend/src/navigations/TabBar.tsx:88]: Here is the preview bottom 7 and margin horizontal a little wider

Comment (rushiii3) [reply frontend/src/navigations/TabBar.tsx:88]: > Reduce Bottom to 0 or 6-7 range.
> 
> Set the margin horizontal in a way that the screen becomes within the bottom tab.
> 
> Reference:
> 
> image
> 
> But don't make the icon black and white, they look nice.

Is this okay ? need changes?

Comment (rushiii3) [reply frontend/src/navigations/TabBar.tsx:88]: Check this one out !

Comment (SB2318) [reply frontend/src/navigations/TabBar.tsx:88]: set borderBottomLeftRadius & borderBottomRightRadius to 0.

Comment (rushiii3) [reply frontend/src/navigations/TabBar.tsx:88]: This is the preview borderRadius 0 
<image src="https://github.com/SB2318/UltimateHealth/assets/105168088/dd538c7c-de86-4ef2-ac41-f0c2cf722b7c"  height="600"/>

**Review by SB2318 — APPROVED** [review]

**Pull request merged**